src/OpenSage.Game/Data/Utilities/Extensions/BinaryReaderExtensions.cs

```diff
@@ -396,6 +404,28 @@ public static List<T> ReadListAtOffset<T>(this BinaryReader reader, Func<T> crea
             return result;
         }
 
+        public static List<T> ReadFixedSizeListAtOffset<T>(this BinaryReader reader, Func<T> creator, uint size) where T : class
```

Comment:
What's the reason for this, in addition to `ReadFixedSizeArrayAtOffset`?

Reply:
I needed it for the new DefineFunction2. I guess i should decide wether i use Array or List at some point